src/pages/atoms/index.tsx

```diff
@@ -1,33 +1,173 @@
+  });
+
+  const onSubmit = (data: FormData) => {
+    console.log("Form submitted:", data);
+  };
+
   return (
-    <div className="flex flex-col w-full h-screen p-10 bg-[#ECECEC]">
+    <div className="flex flex-col gap-10 w-full min-h-screen p-10 bg-[#ECECEC]">
       <div className="flex flex-row w-full justify-center items-center text-[30px] font-extrabold">
-        <h1 className="text-center mb-6">Components</h1>
+        <h1 className="text-center">Components</h1>
       </div>
-      <Card className="h-[300px] p-6 gap-2 grid grid-cols-2">
+      <Card className="p-6 gap-2 grid grid-cols-2">
+        <span className="col-span-2 text-[24px] font-bold">TextInput</span>
+        <line className="col-span-2 w-full border-b border-black mb-6" />
+        <TextInput
+          label="TextInput"
+          placeholder="Enter text"
+          className="col-span-2 w-[500px]"
+        />
+        <TextInput
+          isRequired
+          label="Required TextInput"
+          placeholder="Enter text"
+          className="col-span-2 w-[500px]"
+        />
+        <TextInput
+          isRequired
+          label="TextInput Error"
+          placeholder="Enter text"
+          className="w-[500px]"
+          error={mockError}
+        />
+      </Card>
+
+      <Card className="p-6 gap-2 grid grid-cols-2">
+        <span className="col-span-2 text-[24px] font-bold">PasswordInput</span>
+        <line className="col-span-2 w-full border-b border-black mb-6" />
+        <PasswordInput
+          label="PasswordInput"
+          placeholder="Enter Password"
+          className="col-span-2 w-[500px]"
+        />
+        <PasswordInput
+          isRequired
+          label="Required PasswordInput"
+          placeholder="Enter Password"
+          className="col-span-2 w-[500px]"
+        />
+        <PasswordInput
+          isRequired
+          label="PasswordInput Error"
+          className="w-[500px]"
+          error={mockPasswordError}
+        />
+      </Card>
+
+      <Card className="h-full p-6 gap-2 grid grid-cols-2">
+        <span className="col-span-2 text-[24px] font-bold">TextArea</span>
+        <line className="col-span-2 w-full border-b border-black mb-6" />
+        <TextArea
+          label="TextArea"
+          placeholder="Enter text"
+          className="col-span-2 w-[500px]"
+        />
+        <TextArea
+          label="TextArea"
+          placeholder="Enter text"
+          isRequired
+          className="col-span-2 w-[500px]"
+        />
+        <TextArea
+          isRequired
+          label="TextArea Error"
+          className="w-[500px]"
+          height="200px"
+          placeholder="Enter text"
+          error={mockError}
+        />
+      </Card>
+
+      <Card className="h-full p-6 gap-2 grid grid-cols-2">
+        <span className="col-span-2 text-[24px] font-bold">EmailInput</span>
+        <line className="col-span-2 w-full border-b border-black mb-6" />
+
+        <form
```

Comment:
May be There is some problem while checking the email input feild. When I am giving a valid email id and clicked submit then also it is showing Requiered .